code/src/testResources/pcgen/lang/cleaned.properties

```diff
@@ -93,16 +90,14 @@ in_InfoPrepared=Prepared Spells
 
 in_InfoSpellbooks=Spell Books
 
-in_races=Races
-
 in_Templates=Templates
 
 #Other Universal stuff
 in_sourceLabel=Source
 
 #Other Universal stuff
 
-in_template=Template
+in_condition=Condition
```

Comment:
Was two entries, and now it's one?

Reply:
We don't have `in_template` at all anymore